control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -93,7 +93,7 @@ type Reconciler struct {
 	ServiceLister      corelisters.ServiceLister
 	SubscriptionLister messaginglisters.SubscriptionLister
 
-	Prober prober.Prober
+	NewProber prober.NewProber
```

Comment:
```suggestion
	Prober prober.NewProber
```
We should still call this `Prober`